tools/clang/test/HLSL/workgraph/work-graphs.hlsl

```diff
@@ -308,3 +308,24 @@ void node4_06(RWThreadNodeInputRecord<RECORD> input)
 {
   input.FinishedCrossGroupSharing(); // expected-error {{no member named 'FinishedCrossGroupSharing' in 'RWThreadNodeInputRecord<RECORD>'}}
 }
+
+struct R
+{
+  uint3 dtid: SV_DispatchThreadID;
```

Comment:
Is there a document somewhere that describes what system values are supported for node shaders? The preview at https://devblogs.microsoft.com/directx/d3d12-work-graphs-preview wasn't explicit, but I did see SV_DispatchGrid that isn't in this list.

Reply:
This PR is only for common input before shader model 6.8.
So SV_DispatchGrid is not here.

Reply:
SV_DispatchGrid is also different.  It is a special record field, instead of a system-generated input for the shader separate from the record.  It is also the only recognized system value inside a record (for now).

Reply:
The legal non-record/node object system value inputs for node shaders are described [here](https://microsoft.github.io/DirectX-Specs/d3d/WorkGraphs.html#node-shader-system-values).